### PR TITLE
DOCS-12907 revision to VM/BM endpoint instructions

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc [hunk 1]
```diff
@@ -71,21 +73,21 @@ image::rtf-ingress-endpoint.png[A host and path are set for the application's en
...
 
 [NOTE]
 To access a Runtime Fabric instance using more than one DNS, add additional DNS entries in the SAN certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Application url* section.
 
-To specify network configuration for the application:
-
-. Click the *Ingress* tab.
-. Enable or disable the following options:
+. Select *Ingress*.
+. From the *Domain* drop-down list, choose the appropriate domain.
```
Comment (muleysteve) [inline modules/ROOT/pages/deploy-to-runtime-fabric.adoc:87]:
```suggestion
. Enable the *Enable inbound traffic* checkbox.
. From the *Domain* drop-down list, choose the appropriate domain.
```

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc [hunk 2]
```diff
@@ -71,21 +73,21 @@ image::rtf-ingress-endpoint.png[A host and path are set for the application's en
...
 +
-* *Enable inbound traffic to this application*: Allow incoming HTTP requests to be processed by this application. Ensure your application is configured to accept connections on port 8081.
+. Enable or disable the following options:
```
Comment (muleysteve) [inline modules/ROOT/pages/deploy-to-runtime-fabric.adoc]:
```suggestion
. Optionally, enable or disable the following options:
```

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc [hunk 3]
```diff
@@ -71,21 +73,21 @@ image::rtf-ingress-endpoint.png[A host and path are set for the application's en
...
-=== Specify the Network Configuration for the Application (for VMs/Bare Metal)
+=== Configure a Public Endpoint for the Application for VMs/Bare Metal
+
+Use the following steps to configure a public endpoint and network configuration for an application deployed to Runtime Fabric for VMs/Bare Metal.
 
-If you are using Runtime Fabric on VMs/Bare Metal, you must xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. 
+You must first xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. Ensure your application is configured to accept connections on port 8081.
 
 [NOTE]
 To access a Runtime Fabric instance using more than one DNS, add additional DNS entries in the SAN certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Application url* section.
```
Comment (muleysteve) [inline modules/ROOT/pages/deploy-to-runtime-fabric.adoc]:
```suggestion
To access a Runtime Fabric instance using more than one FQDN, add additional DNS entries in the SubjectAltName (SAN) certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Public endpoint* section. Each deployed application can only be addressed using a single FQDN.
```

**Review by muleysteve — COMMENTED** [review]
Adjustments for consistency with product UI.

File: modules/ROOT/pages/deploy-to-runtime-fabric.adoc [hunk 4]
```diff
@@ -71,21 +73,22 @@ image::rtf-ingress-endpoint.png[A host and path are set for the application's en
...
 
-=== Specify the Network Configuration for the Application (for VMs/Bare Metal)
+=== Configure a Public Endpoint for the Application for VMs/Bare Metal
 
-If you are using Runtime Fabric on VMs/Bare Metal, you must xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. 
+Use the following steps to configure a public endpoint and network configuration for an application deployed to Runtime Fabric for VMs/Bare Metal.
 
-[NOTE]
-To access a Runtime Fabric instance using more than one DNS, add additional DNS entries in the SAN certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Application url* section.
+You must first xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. Ensure your application is configured to accept connections on port 8081.
```
Comment (luanamulesoft) [inline modules/ROOT/pages/deploy-to-runtime-fabric.adoc:80]: You must first xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. Ensure that your application is configured to accept connections on port 8081.

**Review by luanamulesoft — APPROVED** [review]

**Review by luanamulesoft — APPROVED** [review]

**Pull request merged**